src/components/App.tsx

```diff
@@ -1046,7 +1042,7 @@ export class App extends React.Component<any, AppState> {
             {
               // we need to recreate the element to update dimensions &
               //  position
-              ...newTextElement(element, text, element.font),
+              ...newTextElement({ ...element, text, font: element.font }),
             },
```

Comment:
Could this just be `newTextElement({ ...element, text, font: element.font })`?

Reply:
Good catch :+1: